recipes/streamtracer/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "streamtracer" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamtracer-{{ version }}.tar.gz
+  sha256: 29955653f88970ab3b1fe3e9c470e3b243c1a04e157647d22b2b13494f8864ca
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - numpy
+    - maturin >=0.13,<0.14
+    - pip
+  run:
+    - python
+    - numpy >=1.14.5
```

B/c numpy is used during the build you need this here.

```suggestion
    - {{ pin_compatible('numpy') }}
```